core/src/main/scala/kanaloa/reactive/dispatcher/queue/QueueProcessor.scala

```diff
@@ -2,10 +2,9 @@ package kanaloa.reactive.dispatcher.queue
 
 import akka.actor._
 import kanaloa.reactive.dispatcher.ApiProtocol._
-import kanaloa.reactive.dispatcher.metrics.Metric
+import kanaloa.reactive.dispatcher.metrics.{MetricsCollector, Metric}
 import kanaloa.reactive.dispatcher.queue.Queue.Retire
 import kanaloa.reactive.dispatcher.queue.QueueProcessor._
-import kanaloa.reactive.dispatcher.queue.Worker.Hold
 import kanaloa.reactive.dispatcher.{Backend, ResultChecker}
 import kanaloa.util.FiniteCollection._
```

Comment:
unused..and this entire class can probably be removed, since we are not using WorkHistory any more